FlowCrypt/Controllers/Bootstrap/InvalidStorageViewController.swift

```diff
@@ -6,9 +6,18 @@
 //  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
 //
 
-import UIKit
+import FlowCryptUI
+import AsyncDisplayKit
+
+@MainActor
```

Comment:
`TableNodeViewController` already has `@MainActor` annotation, inheritance should work here.